target/linux/ath79/dts/qca9563_tplink_archer-a6-v2.dts

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9563_tplink_archer-x6-v2.dtsi"
+
+/ {
+	compatible = "tplink,archer-a6-v2", "qca,qca9563";
+	model = "TP-Link Archer A6 v2";
```

Comment:
You can think about changing this to:
model = "TP-Link Archer A6 v2/C6 v2 (US)";

Reply:
done.